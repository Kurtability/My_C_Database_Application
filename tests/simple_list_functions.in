list keys
list entries
set a 1
list keys
set b 2
list keys
set c 3
list keys
list entries
list snapshots
bye
