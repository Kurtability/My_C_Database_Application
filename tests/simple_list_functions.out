> list keys
no keys

> list entries 
no entries

> set a 1 
ok

> list keys
a

> set b 2
ok

> list keys
b
a

> set c 3 
ok

> list keys 
c
b
a

> list entries 
c [3]
b [2]
a [1]

> list snapshots 
no snapshots

> bye
bye
